doc/classes/Viewport.xml

```diff
@@ -407,45 +407,48 @@
 		<constant name="DEBUG_DRAW_VOXEL_GI_EMISSION" value="8" enum="DebugDraw">
 			Objects are displayed with only the emission color from [VoxelGI]s.
 		</constant>
-		<constant name="DEBUG_DRAW_SHADOW_ATLAS" value="9" enum="DebugDraw">
+		<constant name="DEBUG_DRAW_UV2_TEXEL_DENSITY" value="9" enum="DebugDraw">
+			Objects are displayed with a checkerboard pattern indicative of their UV2 texel density.
```

Comment:
We should mention here how wide (in texels) the squares are. If I read your code correctly then users can expect that each square is 10 texels by 10 texels